lms/static/scripts/frontend_apps/index.js

```diff
@@ -35,6 +35,16 @@ export function init() {
    */
   const services = new Map();
 
+  if (config?.debug?.values && Object.keys(config.debug.values).length) {
```

Comment:
```suggestion
  if (config.debug?.values && Object.keys(config.debug.values).length) {
```

The `readConfig` call above will either return a `config` object or throw.